### ISSUE TITLE
Remove RPI workflow

### ISSUE THREAD

**Issue by rolznz** [getAlby/nostr-wallet-connect-next#492]
From @rdmitr in https://github.com/getAlby/nostr-wallet-connect-next/pull/480

> I have updated the http.yml action to build packages for x86_64, ARM v6, AARCH64. Haven't had a chance to test the ARM v6 package on Raspberry Pi yet, though. If, however, it works fine, we can retire the package-raspberry-pi.yml action and only use this one.

I also did not test RPI yet.

**Comment by rolznz** [comment]
Are there any downsides to consider here and is it worth doing? will the file downloads be bigger? CC @reneaaron

**Comment by rdmitr** [comment]
No, it's going to be the same package, built the same way. That's why I suggested merging desktop and RPi actions — they do essentially the same, the only difference is the cross-compiler used for RPi

**Comment by rdmitr** [comment]
The Raspberry Pi action is to be removed by https://github.com/getAlby/nostr-wallet-connect-next/pull/480

**Comment by rdmitr** [comment]
@rolznz I did a quick test here — the app seems to run fine on RPi 👍
